test/clj/rems/test/db/workflow.clj

```diff
@@ -1,19 +0,0 @@
-(ns rems.test.db.workflow
```

Comment:
why remove these three test files?

Reply:
They were duplicating each other. The common functionality has been moved to https://github.com/CSCfi/rems/blob/1828404b0506a86462d3175e83a22cc8bc1e4080/src/clj/rems/db/core.clj#L17 https://github.com/CSCfi/rems/blob/1828404b0506a86462d3175e83a22cc8bc1e4080/test/clj/rems/test/db/core.clj#L36